Automattic/woocommerce-payments
# Add AuthorizationRequired State

Fixes #7418 

#### Changes proposed in this Pull Request

This PR adds Authentication required state that handles the situation when payment needs to be authorized (3DS). Among others, this PR introduces multiple state changes.







#### Testing instructions



* Make sure that all tests passed
* Make sure to test with a credit card that requires 3DS (4000000000003063)
* Make sure that you are redirected to the authorization URL.

-------------------


**Post merge**
